holoviews/tests/core/data/test_spatialpandas.py

```diff
@@ -301,3 +304,14 @@ def test_dict_dataset_add_dimension_values(self):
 
     def test_sort_by_value(self):
         raise SkipTest("Not supported")
+
+
+
+def test_regression_get_value_array():
+    # See: https://github.com/holoviz/holoviews/pull/6519
```

Comment:
Do you prefer to link to the PR or to the issue?

Reply:
It's definitely a PR. A good PR should contain all the relevant information. I have updated the main post with more  information.